### PR TITLE
LPS-104540 Athanasius

### PR DESCRIPTION
**Pull request by petershin**

### CODE REVIEW

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:test:bundle

**Comment by petershin** [comment]
ci:test:gradle-plugins

**Comment by petershin** [comment]
ci:test:project-templates

**Comment by petershin** [comment]
ci:test

**Comment by jpince** [comment]
@petershin startup errors, the last PR had similar.
```
Failed to install artifact: /opt/dev/projects/github/liferay-portal/bundles/osgi/war/classic-theme.war 
java.lang.RuntimeException: com.liferay.portal.kernel.deploy.auto.AutoDeployException: com.liferay.portal.kernel.deploy.auto.AutoDeployException: classic-theme.war does not support this version of Liferay
```
If you need help, it's probably better to ask in #t-frontend-infra, as I'm not sure of Chema's availability this week due to Italian Symposium & DEVCON.

File: release.properties [hunk 1]
```diff
@@ -53,11 +53,11 @@
...
     #release.info=on
 
-    release.info.build=7201
+    release.info.build=7300
     release.info.date=November 12, 2019
     release.info.name=Liferay Community Edition Portal
-    release.info.version=7.2.1
-    release.info.version.display.name=7.2.1 CE GA2
+    release.info.version=7.3.0
+    release.info.version.display.name=7.3.0 CE M1
```
Comment (jpince) [inline release.properties]: Please change to `7.3.0 CE GA1`. We won't have pre-releases for 7.3 (a lot of GAs instead)

**Comment by petershin** [comment]
@jpince - Do you know if anyone is updating the definition files to `7.3`?

**Comment by petershin** [comment]
ci:test:sf

**Comment by petershin** [comment]
ci:test:bundle

**Comment by petershin** [comment]
ci:test:gradle-plugins

**Comment by petershin** [comment]
ci:test:project-templates

**Comment by petershin** [comment]
ci:test

**Comment by petershin** [comment]
ci:test:gradle-plugins

**Pull request closed without merging**